lock/lockmock/lock.go

```diff
@@ -1,11 +1,13 @@
-package lock
+package lockmock
 
 import (
 	"sync"
 	"time"
+
+	lock "github.com/Scalingo/go-etcd-lock/lock"
 )
 
-type MockLocker struct {
+type LockerMock struct {
```

Comment:
You're renaming this mock because of the `gomock` mock taking the name?

Could you use something like `MockLockerMutex` ? for this one ?

Reply:
Actually I wanted to have the same naming scheme than in the package `nsqproducermock` in go-internal-tools (`ProducerMock` and `MockProducer`). But it's OK to rename it to `MockLockerMutex` if you prefer.